CvGameCoreDLL/CvPlayerAI.cpp

```diff
@@ -4978,10 +4978,10 @@ TechTypes CvPlayerAI::AI_bestTech(int iMaxPathLength, bool bIgnoreCost, bool bAs
 									iValue /= 2;
 								}
 
-								/*if (iI == ASTRONOMY || iI == COMPASS) { //otherwise they tend to beeline

```

Comment:
Have you checked if Spain/Portugal still make it to the new world reasonably fast?

Reply:
I'll check with some Inca/Aztec test games. Spain and Portugal still have massive inclination for researching Cartography, Exploration & Optics (lines 5182 and 5289, respectively).

Reply:
I just posted the results of my test games in the forum. Should I make further changes to the tech priority code or is it fine for now?

Reply:
I made further tweaks. AI Spain seems to be consistently ahead in the race for the new world. Portugal, however, can't catch up, even after letting them build more settlers after getting the tech for their UU. They instead prefer to wait for galleons before sending settlers to Brazil.